libr/include/r_core.h

```diff
@@ -497,6 +497,7 @@ R_API RList *r_core_get_func_args(RCore *core, const char *func_name);
 R_API void r_core_print_func_args(RCore *core);
 R_API char *resolve_fcn_name(RAnal *anal, const char * func_name);
 
+R_API void save_parsed_type(RCore *core, const char *parsed);
```

Comment:
if you need it from cutter. find a good name, same goes for that shitty resolve_fcn_name defined above

Reply:
resolve_fcn_name was the reason I though r_core_ prefix was not required. Sorry for that. Will rename to r_core_save_parsed_type